OpenOrchestrator/common/db_util.py

```diff
@@ -625,18 +625,21 @@ def get_next_single_trigger() -> list[str]:
     )
 
     trigger = conn.execute(command).fetchone()
-    return list[trigger]
+    if trigger is not None:
+        return list(trigger)
+
+    return None
```

Comment:
I would probably write this as 

```suggestion
    return None if trigger is None else list(trigger)
```

or 

```suggestion
    return list(trigger) if trigger is not None else None
```

but I'm not sure if any of these are more readable than your code …

Reply:
I think your suggestion makes the code too dense, but I understand where you're coming from